ui/app/common/domain/services/programService.js

```diff
@@ -121,6 +121,11 @@ angular.module('bahmni.common.domain')
             });
         };
 
+        var disableProgramOutcome = function () {
+            var config = appService.getAppDescriptor().getConfigValue('disableProgramOutcome');
+            return config ? config : false;
```

Comment:
Can we directly do 
`var disableProgramOutcome = function () {
          return appService.getAppDescriptor().getConfigValue('disableProgramOutcome') || false;
}`